Firestore/core/src/firebase/firestore/util/warnings.h

```diff
@@ -42,48 +42,50 @@
 
 #define SUPPRESS_BEGIN_(name) _Pragma("GCC diagnostic push") _Pragma(name)
 
+#define SUPPRESS_END() _Pragma("GCC diagnostic pop")
+
 #elif defined(_MSC_VER)
 #define SUPPRESS_BEGIN_UNIMPLEMENTED_() __pragma(warning(push))
 
 #define SUPPRESS_BEGIN_(name) \
   __pragma(warning(push)) __pragma(warning(disable : name))
 
+#define SUPPRESS_END() __pragma(warning(pop))
+
+#else:
```

Comment:
What does the trailing `:` do?

Reply:
It indicates that I've been doing a bit too much python programming lately... Fixed.